### PR TITLE
Add tons of food to the various mansion store rooms

### PR DESCRIPTION
**Pull request by theboo**
Feel free to tone down rolls, or add/remove items from here xD may have gone a bit overboard.. Tho, in a pack where eternal steak exists i dont think it matters much.. Here's everything from a single storeroom. As well as another pic of the amount of chests in a store room.

image
image

### CODE REVIEW

**Comment by RobuxShooters** [comment]
This feels like a good change. It always felt so underwhelming to come across these chests but only get a bone or two from each chest.

**Comment by NillerMedDild** [comment]
I think the rolls could stand to be reduced to 1 min, 3 max, the amount you have there is slightly insane 😄

File: kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/repurposed_structures/mansion.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                    weight: 50,
+                    count: [4, 16]
+                },
+                {
+                    item: 'create:builders_tea',
+                    weight: 10,
+                    count: [4, 16]
+                },
+                {
+                    item: 'sushigocrafting:onigiri',
```
Comment (MuteTiefling) [inline kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/repurposed_structures/mansion.js:102]: missing nbt `Item.of('sushigocrafting:onigiri', '{Weights:[I;0,0],Spices:{}}')`

File: kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/repurposed_structures/mansion.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    weight: 50,
+                    count: [2, 8]
+                },
+                {
+                    item: 'simplefarming:sake',
+                    weight: 40,
+                    count: [1, 2]
+                },
+                {
+                    item: 'sushigocrafting:tobiko_tuna_cucumber_california',
```
Comment (MuteTiefling) [inline kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/repurposed_structures/mansion.js:47]: `Item.of('sushigocrafting:tobiko_tuna_cucumber_california', '{Weights:[I;0,0,0,0,0,0],Spices:{}}')`

**Review by MuteTiefling — COMMENTED** [review]
Looks good. Think Niller's right, too many rolls per chest, given how many chests there are. Also the sushigo stuff needs NBT to actually give benefits.

**Comment by theboo** [comment]
fixed sushi stuff, went with 2-3 rolls instead of 1-3 as 1 roll left the chests feeling a bit sad to open. here's current output from one storage room:

image

**Comment by ZephyrWindSpirit** [comment]
How do you make drinks in simple farming? JEI doesn't tell you.

**Comment by MuteTiefling** [comment]
> How do you make drinks in simple farming? JEI doesn't tell you. 

https://github.com/cweckerl/simplefarming/wiki/Brewing-Information

**Review by NillerMedDild — APPROVED** [review]

**Pull request merged**